### PR TITLE
Update FaceSentimentAnalyzer sample and apps to new version of interface

### PR DESCRIPTION
**Pull request by LPBourret**
Update FaceSentimentAnalyzer sample and apps to new version of Microsoft.AI.Skills.SkillInterfacePreview (0.6.0) and new versions of C++Winrt (2.0+)

### CODE REVIEW

File: samples/Scripts/AppWRLCPP_PreBuild.ps1 [hunk 1]
```diff
@@ -24,10 +25,43 @@ $OutDir = "$ProjectDir" + "inc"
...
+        mkdir $GeneratedFilesDirFullPath
+	    
+    }
+    else
+    {
+		echo "removing content from $GeneratedFilesDirFullPath"
+        Get-ChildItem -Path '$GeneratedFilesDirFullPath' -Recurse | Remove-Item -force -recurse
+    }
+	
+	$DestinationPath = $GeneratedFilesDirFullPath + "winrt\"
```
Comment (Sudhaushu) [inline samples/Scripts/AppWRLCPP_PreBuild.ps1]: >winrt [](start = 50, length = 5)

These are not winrt headers.. maybe we call it "wrl\" or just "inc"

Comment (LPBourret) [reply samples/Scripts/AppWRLCPP_PreBuild.ps1]: for the script only, headers are generated by midl in that folder

File: samples/Scripts/AppWRLCPP_PreBuild.ps1 [hunk 2]
```diff
@@ -45,8 +80,20 @@ foreach ($file in $idlfilelist)
...
     Invoke-Expression "$headergencmd"
 }
+
+if($GeneratedFilesDir -ne "")
+{
+    $filesToCopy = Get-ChildItem -Path "." -Filter *.h -File | %{$_.FullName}
+    foreach($file in $filesToCopy)
+    {
+        echo "copying $file to $DestinationPath"
+	    Copy-Item $file -Destination $DestinationPath
```
Comment (Sudhaushu) [inline samples/Scripts/AppWRLCPP_PreBuild.ps1]: >Copy-Item $file -Destination $DestinationPath [](start = 5, length = 45)

why do we need this step?

File: samples/Scripts/AppWRLCPP_PreBuild.ps1 (hunk 1 above)
Comment (Sudhaushu) [reply samples/Scripts/AppWRLCPP_PreBuild.ps1]: these headers are not winrt though.

---
In reply to: [316925564](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/53#discussion_r316925564) [](ancestors = 316925564)

**Pull request merged**